posts/test/test3.md

```diff
@@ -0,0 +1,14 @@
+---
+media:
+  - slac
```

Comment:
It's called slack :-)

Reply:
I'm just testing what if it is mispronounced! :)

Reply:
Awesome!